guides/source/association_basics.md

```diff
@@ -117,16 +117,22 @@ class CreateBooks < ActiveRecord::Migration[5.0]
 end
 ```
 
+When used alone, `belongs_to` produces a one-directional one-to-one connection. In the example above - while each book "knows" its author, the latter model does not know anything about it's books.
+To setup a [bi-directional association](#bi-directional-associations) - use `belongs_to` in combination with a `has_one` or `has_many` on the other model. 
+
 ### The `has_one` Association
 
-A `has_one` association also sets up a one-to-one connection with another model, but with somewhat different semantics (and consequences). This association indicates that each instance of a model contains or possesses one instance of another model. For example, if each supplier in your application has only one account, you'd declare the supplier model like this:
+A `has_one` association indicates that each instance of a model contains or possesses one instance of another model.
```

Comment:
A `has_one` association indicates that one other model has a `belongs_to` reference to this model. That model can be fetched through this association.